database/insertData.js

```diff
@@ -1,12 +1,35 @@
 const fs = require('fs')
+const path = require('path')
 const fetch = require('node-fetch')
 const config = require('../src/config.json')
 
+const dataFolder = './database/insertData'
+const sharedDataFiles = ['core_mutations.js', 'dev_actions.js']
+
 if (process.argv[2] === '--from_insert_data.sh') {
   const filesToProcess = fs
-    .readdirSync('./database/insertData')
+    .readdirSync(dataFolder, { withFileTypes: true })
+    .filter((dirent) => dirent.isFile())
+    .map((dirent) => dirent.name)
     .filter((file) => !file.match(/^\./)) // Ignore hidden files
-    .filter((file) => !['core_mutations.js', 'dev_actions.js'].includes(file))
+    .filter((file) => !sharedDataFiles.includes(file))
+
+  // Add locale-specific files
+  const localeFolder = process.argv[3]
```

Comment:
can this default to `dev` by default, if not supplied ?

Reply:
Yes! 2204d26b